redis/src/aio/mod.rs

```diff
@@ -140,6 +140,20 @@ where
         }
     }
 
+    if connection_info.client_name.is_some() {
+        match cmd("CLIENT")
+            .arg("SETNAME")
+            .arg(connection_info.client_name.as_ref().unwrap())
```

Comment:
instead of using unwrap, you can use an `if let` expression.

Reply:
show me please.
Im pretty sure I took this line from other command issuing instance

Reply:
will do if let